verl/trainer/ppo/core_algos.py

```diff
@@ -719,17 +719,15 @@ def agg_loss(loss_mat: torch.Tensor, loss_mask: torch.Tensor, loss_agg_mode: str
     return loss
 
 
-def compute_policy_loss(
-    old_log_prob,
-    log_prob,
-    advantages,
-    response_mask,
-    cliprange=None,
-    cliprange_low=None,
-    cliprange_high=None,
-    clip_ratio_c=3.0,
```

Comment:
despite that verl does not list `compute_policy_loss` as an official API, we do notice that many verl extensions import this function in their recipe. I'd recommend mark it with `verl.utils.import_utils.deprecated`, add a new function compute_policy_loss_vanilla, note it in https://github.com/volcengine/verl/discussions/2744. We then remove compute_policy_loss in the next release.